.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.4']
+        wordpress: ['4.7', 'latest-stable', 'latest-stable (multisite)', 'nightly']
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: wordpress
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          extensions: mbstring, intl, redis, pdo_mysql
+          php-version: ${{ matrix.php }}
+      - name: Start MySQL
+        run: sudo service mysql start
```

Comment:
This is the mysql instance on the runner (which we probably shouldn't use because we have no control over its version), not the one in the container added above 🙂

Reply:
Urgh, right, sorry. 😓

Reply:
@aaemnnosttv the version of MySQL doesn't matter if it is 5.6 or higher. Why do you want to control its version?